services/course.js

```diff
@@ -4,11 +4,11 @@ const { DOCUMENT_NOT_FOUND } = require('~/consts/errors')
 const { createError, createForbiddenError } = require('~/utils/errorsHelper')
 
 const courseService = {
-  getCourses: async ({ author, skip, limit }) => {
+  getCourses: async ({ author, skip, limit, sort }) => {

```

Comment:
keep in mind that for now getCourses accepts arguments without destructuring (match, skip, limit). What do you think maybe try use getSortOptions fn for sort and pass sortOptions instead ?